samples/reactjs-todo/client/components/icons/apple-icon.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * forgerock-sample-web-react
+ *
+ * apple-icon.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import React from 'react';
+
+/**
+ * @function AppleIcon - React component for the user icon representing the apple icon
+ * @param {Object} props - React props object
+ * @param {string} props.classes - A string of classnames to be set on component
+ * @param {string} props.size - A string representing the intended size of the rendering
+ * @returns {Object} - React JSX Object
+ */
+export default function AppleIcon({ classes = '', size = '24px' }) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      className={classes}
+      height={size}
+      width={size}
+      viewBox="0 0 256 256"
+    >
+      <g
+        fill="#070000"
+        strokeMiterlimit="10"
+        fontFamily="none"
+        fontSize="none"
+        fontWeight="none"
+        textAnchor="none"
+        style={{ mixBlendMode: 'normal' }}
```

Comment:
I believe all these properties can be removed, yeah?

Reply:
removing it breaks the svg 
<img width="510" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/ef3fed5b-becb-4ec9-b72d-c065e64014ad">

Reply:
Which of the above properties are required? I wouldn't think al the `font*` related properties are required, yeah? We can keep the ones required, but trim the ones that aren't.

Reply:
done